index.js

```diff
@@ -243,7 +235,7 @@ async function main(filePaths) {
     const sortedFiles = await getSortedFilePaths(filePathsFromTruffleRoot);
     await printContactenation(sortedFiles);
   } catch (error) {
-    console.log(error);
+    console.log(error, error.stack);
```

Comment:
This is usually helpful for me, but I'm not a zealot of it as I've seen it pollute logs

Reply:
Sounds good!